### PR TITLE
dev-util/poke: bump to version 2.1

### PR DESCRIPTION
**Pull request by kernlpanic**
Closes: https://bugs.gentoo.org/828835
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Dennis Eisele <kernlpanic@dennis-eisele.de>

### CODE REVIEW

File: dev-util/poke/poke-1.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	virtual/pkgconfig
+"
+
+src_configure() {
+	econf \
+		--disable-gui \
+		--disable-libnbd \
+		$(use_enable machine-interface mi) \
+		$(use_enable nls) \
+		$(use_enable static-libs static)
```
Comment (thesamesam) [inline dev-util/poke/poke-1.4.ebuild:42]: I suspect we don't actually want/need static-libs for this at all.

Comment (thesamesam) [reply dev-util/poke/poke-1.4.ebuild:42]: (If you drop the line and simplify src_install, it should be fine w/ EAPI 8 which passes --disable-static)

**Comment by mid-kid** [comment]
The current version of poke is 2.1. I don't think adding an ebuild for 1.4 isn't very useful at this point.

**Comment by kernlpanic** [comment]
> The current version of poke is 2.1. I don't think adding an ebuild for 1.4 is very useful at this point.

Thanks for the feedback you are right. I changed the new version to 2.1

File: dev-util/poke/poke-1.4.ebuild (hunk 1 above)
Comment (kernlpanic) [reply dev-util/poke/poke-1.4.ebuild]: Thanks for the feedback. I fixed that

**Comment by thesamesam** [comment]
Thanks! See 4a590d047fee354e2ca2b7e5a521ee8c9bb170e4 for some style changes.

**Comment by kernlpanic** [comment]
Thank you for merging and for applying the style changes :) I'll take a better look next time before submitting

**Pull request closed without merging**